src/ducks/lagret-filter-ui-state.ts

```diff
@@ -0,0 +1,111 @@
+import {OrNothing} from "../utils/types/types";
+import {LagretFilter} from "./lagretFilter";
+
+// Actions
+export const VELG_LAGRET_FILTER = 'lagretfilter_velg/VELG_LAGRET_FILTER';
+export const MARKER_LAGRET_FILTER = 'lagretfilter_velg/MARKER_LAGRET_FILTER';
+export const AVMARKER_LAGRET_FILTER = 'lagretfilter_velg/AVMARKER_LAGRET_FILTER';
+export const AVMARKER_SISTE_VALGT_LAGRET_FILTER = 'lagretfilter_velg/AVMARKER_SISTE_VALGT_FILTER';
+
+export const MARKER_VEILEDER_GRUPPE = 'lagretfilter_velg/MARKER_VEILEDER_GRUPPE';
+export const AVMARKER_VEILEDER_GRUPPE = 'lagretfilter_velg/AVMARKER_VEILEDER_GRUPPE';
+
+export const APEN_LAGRE_FILTER_MODAL = 'lagretfilter_velg/APEN_LAGRE_FILTER_MODAL';
```

Comment:
And this is either Mine filter modal or Veiledergruppe modal.. need to know which one it is

Reply:
aha, this one is Mine filter modal...so I will rename it